HealthMate/HealthMate.API/Validators/ShortActivityTypeViewModelValidator.cs

```diff
@@ -0,0 +1,29 @@
+﻿using FluentValidation;
+using HealthMate.API.ViewModels.ActivityType;
+
+namespace HealthMate.API.Validators
+{
+    public class ShortActivityTypeViewModelValidator : AbstractValidator<ShortActivityTypeViewModel>
+    {
+        public ShortActivityTypeViewModelValidator()
+        {
+            RuleFor(a => a.Name)
+                .NotEmpty()
+                .NotNull()
+                .Matches("^[a-zA-Z ]*$")
+                .WithMessage("Activity type name can only contain letters and spaces.")
+                .MinimumLength(3)
+                .MaximumLength(100)
+                .Must(StartWithUppercase)
+                .WithMessage("Activity type name must start with an uppercase letter.");
+        }
+
+        private bool StartWithUppercase(string name)
+        {
```

Comment:
You can create extension method and avoid duplication of this private method in several classes